### PR TITLE
feat: Quick Control Panel for the flyPad Status bar

### PR DESCRIPTION
**Pull request by frankkopp**
## Summary of Changes
Introduces a Quick Control panel open from the flyPad Status Bar to give access for regularly used settings and quick actions. 

- [x] Basic functionality
- [x] Create Design and Layout
- [x] Implement design/layout

This PR also contains smaller fixes and changes that were necessary for the quick actions to work:

- Changed ADIRS alignment logic in `src/systems/systems/src/navigation/adirs.rs` to check if setting has been change to INSTANT to allow instant alignment after alignment has started.
- Improved handling of changes to settings of the SimBridge Enabled setting (CONFIG_SIMBRIDGE_ENABLED) to allow for instant reset of the connection attempts. 

## Screenshots
image
image

Feature Recording (without Autobrightness)
https://user-images.githubusercontent.com/16833201/209857883-6e0bfb77-727e-474e-9e9a-6f480148aab8.mp4

Discord username (if different from GitHub): Cdr_Maverick#6475

## Testing instructions
Please test the new Quick Controls features - short docu:

- Settings Link
	- Open the flyPad Settings flyPad page
- Sleep Button 
 	- puts the flyPad in Sleep mode - this only blanks the screen but does not reset the flyPad
- Power Button 
 	- turns the flyPad off - this also resets the flypad to its initial state
- Align ADIRS 
	- immediately aligns the ADIRS independent of the configured realism alignment setting
- Finish Boarding
	-  immediately loads all passengers and cargo independent of loading time setting 
- SimBridge
	- Shows
		- <background color> and "OFF" when SimBridge connection is turned permanently off
		- amber and "Connecting..." when the aircraft tries to connect to the SimBridge(max 5min)
		- red and "Not Available" if a connection couldn't be established after 5min
		- green and "Connected" if the connection is established
	- Actions
 		- Click when "Off": turns the SimBridge connection on and the aircraft will start trying to connect
		- Click when "Not Available": restarts attempting to connect to SimBridge (max 5min)
		- Click when "Connected": Turns the connection permanently off
		- Click when "Connecting...": Turns the connection permanently off
- OnScreen Keyboard
	-  Turns the OnScreen Keyboard to show automatically to On or Off
	- When On a click into an input field shows the OnScreen Keyboard automatically
- Brightness Slider
	- Sets brightness
	- Deactivated if Auto Brightness is configures
- Auto Brightness
	- This toggles the auto brightness setting - if on the brightness slider is deactivated 

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Comment by frankkopp** [comment]
Closing in favor of https://github.com/flybywiresim/a32nx/pull/7669

File: src/instruments/src/EFB/StatusBar/QuickControls.tsx [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+        case SimBridgeState.CONNECTED: return t('QuickControls.SimBridgeConnected');
+        case SimBridgeState.CONNECTING: return t('QuickControls.SimBridgeConnecting');
+        case SimBridgeState.OFFLINE: return t('QuickControls.SimBridgeOffline');
+        default: return t('QuickControls.SimBridgeOff');
+        }
+    };
+
+    const handleSettings = () => {
+        history.push('/settings/flypad');
+    };
```
Comment (2hwk) [inline src/instruments/src/EFB/StatusBar/QuickControls.tsx:121]: These functions could be memoized with React.memo(), especially ones that are often used

**Review by 2hwk — CHANGES_REQUESTED** [review]

File: src/instruments/src/EFB/StatusBar/QuickControls.tsx (hunk 1 above)
Comment (frankkopp) [reply src/instruments/src/EFB/StatusBar/QuickControls.tsx:121]: > These functions could be memoized with React.memo(), especially ones that are often used

Great tip!
I have done the last three function with use memo now - the others are action handlers so I do not think it makes sense for them.

**Review by Benjozork — APPROVED** [review]

**Comment by mico975** [comment]
Quality Assurance Tester Report

Discord          : mico#3145
Object of testing: #7669
Tier of Testing  : 1
Date             : 11/01/2023

Testing Process: 
Please test the new Quick Controls features - short docu:

Settings Link
Open the flyPad Settings flyPad page - **OK**

Sleep Button
puts the flyPad in Sleep mode - this only blanks the screen but does not reset the flyPad - **OK**

Power Button
turns the flyPad off - this also resets the flypad to its initial state - **OK**

Align ADIRS
immediately aligns the ADIRS independent of the configured realism alignment setting - **OK**

Finish Boarding
immediately loads all passengers and cargo independent of loading time setting - **OK**

SimBridge
Shows
and "OFF" when SimBridge connection is turned permanently off - **OK**
amber and "Connecting..." when the aircraft tries to connect to the SimBridge(max 5min) - **OK**
red and "Not Available" if a connection couldn't be established after 5min - **OK**
green and "Connected" if the connection is established - **OK**
Actions
Click when "Off": turns the SimBridge connection on and the aircraft will start trying to connect - **OK**
Click when "Not Available": restarts attempting to connect to SimBridge (max 5min) - **OK**
Click when "Connected": Turns the connection permanently off - **OK**
Click when "Connecting...": Turns the connection permanently off - **OK**

OnScreen Keyboard
Turns the OnScreen Keyboard to show automatically to On or Off - **OK**
When On a click into an input field shows the OnScreen Keyboard automatically - **OK**

Brightness Slider
Sets brightness - **OK**
Deactivated if Auto Brightness is configures - **OK**

Auto Brightness
This toggles the auto brightness setting - if on the brightness slider is deactivated - **OK**

Negatives:       
The power off on the top is counter ituitive because it actually sends FlyPad to sleep instead of full power off.
image

Testing Results: 
Passed

Conclusions: 
LGTM

Media:       
image

**Pull request merged**